rest_lights.cpp

```diff
@@ -2445,7 +2501,14 @@ void DeRestPluginPrivate::handleLightEvent(const Event &e)
                     }
                     else if (item->lastSet().isValid() && (gwWebSocketNotifyAll || (item->lastChanged().isValid() && item->lastChanged() >= lightNode->lastStatePush)))
                     {
-                        state[key] = item->toVariant();
+                        if (rid.suffix == RStateEffect)
+                        {
+                            state[key] = RStateEffectValuesMueller[item->toNumber()];

```

Comment:
Is a guard here for Müller Licht needed?

Reply:
No, but better naming is.  Should probably have used `RStateEffectAllValues` vs `RStateEffectDefaultValues`.  No clue how to expand this to other lights with similar capabilities, but implemented differently.